clarinet/contracts/main-swapr.clar

```diff
@@ -78,63 +91,52 @@
 
 ;; get overall balances for the pair
 (define-public (get-balances (token-x-trait <sip-010-token>) (token-y-trait <sip-010-token>))
-  (let ((token-x (contract-of token-x-trait)) (token-y (contract-of token-y-trait)))
-    (let ((pair (unwrap! (map-get? pairs-data-map { token-x: token-x, token-y: token-y }) invalid-pair-err)))
-      (ok (list (get balance-x pair) (get balance-y pair)))
+  (let 
+    (
+      (token-x (contract-of token-x-trait)) 
+      (token-y (contract-of token-y-trait))
+      (pair (unwrap! (map-get? pairs-data-map { token-x: token-x, token-y: token-y }) invalid-pair-err))
     )
+    (ok (list (get balance-x pair) (get balance-y pair)))
   )
 )
 
 ;; since we can't use a constant to refer to contract address, here what x and y are
```

Comment:
note to self: this 3 lines of comments are no longer relevant, and we should remove them